packages/blade/src/components/Typography/_decisions/decisions.md

```diff
@@ -106,7 +106,7 @@ Another option would be having a `size` prop that would lead us to have a "large
 
   ```jsx
   <Text
-    variant=" medium | large | caption"
```

Comment:
@kamleshchandnani can you confirm if this was expected? In implementation I can see we have `body | caption` in `variant`

Reply:
yes this is correct. forgot to update the RFC after we added body as a variant